hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -350,7 +350,19 @@ boolean processKey(List<String> keyList) throws IOException {
       // TODO RenameKey needs to be changed to batch operation
       for (String key : keyList) {
         String newKeyName = dstKey.concat(key.substring(srcKey.length()));
-        adapter.renameKey(key, newKeyName);
+        try {
+          adapter.renameKey(key, newKeyName);
+        } catch (OMException ome) {
+          LOG.error("rename key failed: {}. source:{}, destin:{}",
```

Comment:
nit: I think the log message can be better.
```
LOG.error("Key rename failed for source key: {} to " +
        "destination key: {}.", key, newKeyName, ome);
``` 
or:
```
LOG.error("Key rename failed with error: {}. Source key: {}, " +
        "Destination key: {}", ome.getMessage(), key, newKeyName);
```

Same for https://github.com/apache/ozone/pull/5253/files#diff-b2802792bb08b569714c90f10ecda42d917dd6329137188472010d2ea7476b1bR344

Reply:
Done.